3rdparty/python/requirements.txt

```diff
@@ -1,7 +1,7 @@
 boto3==1.4.4
 pantsbuild.pants>=1.1.0
 pantsbuild.pants.testinfra>=1.1.0
-pyjavaproperties==0.6
+pyjavaproperties==0.7
 moto==0.4.31
 mock==2.0.0
-six>=1.9.0,<2
\ No newline at end of file
+six>=1.9.0,<2
```

Comment:
lol is it me, or is this line identical?

Reply:
```
~/aiq ❯❯❯ rlwrap ros run
* (string= "six>=1.9.0,<2" "six>=1.9.0,<2")
T
* ^D

~/aiq ❯❯❯ stack ghci
Prelude> "six>=1.9.0,<2" == "six>=1.9.0,<2"
True
Prelude> ^D
Leaving GHCi.

~/aiq ❯❯❯ python3
Python 3.7.5 (default, Nov  1 2019, 02:16:32)
[Clang 11.0.0 (clang-1100.0.33.8)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> "six>=1.9.0,<2" == "six>=1.9.0,<2"
True
```

Hmm, you might be right...